python/lsst/ip/isr/isrStatistics.py

```diff
@@ -348,6 +528,8 @@ def measureCti(self, inputExp, overscans, gains):
             Dictionary of measurements, keyed by amplifier name and
             statistics segment. Everything in units based on electron.
         """
+        self.log.warning("Using the legacy version of CTI statistics may give wrong CTI")
```

Comment:
Do we need a warning here and above?  (Though heck why not, nobody should be running this...)